Server/src/routes/vehicle.route.ts

```diff
@@ -56,7 +56,7 @@ export class VehicleRoute {
 	 */
 	private async getUid(req: Request, res: Response): Promise<void> {
 		const userData: GetUidApp = req.body
-		if (!userData || !userData.trackId || !userData.vehicleName) {
+		if (!userData || userData.trackId == undefined || !userData.vehicleName) {
```

Comment:
This would still disallow `""` (the empty string) as a vehicle name. I am not sure if this is intentional

Reply:
This is rather intentional although we have not really talked about what valid names for vehicles, users, pois, poitypes and vehicletypes area